mne_bids/dig.py

```diff
@@ -206,7 +207,7 @@ def _electrodes_tsv(raw, fname, datatype, overwrite=False, verbose=True):
     if hasattr(raw, 'impedances'):
         data['impedance'] = _get_impedances(raw, names)
 
-    _write_tsv(fname, data, overwrite=overwrite, verbose=verbose)
+    _write_tsv(fname, data, overwrite=True, verbose=verbose)
```

Comment:
I don't think this should be changed…

Reply:
Sorry you're right!

Reply:
Just kidding, I need to actually change this to an analagous error with coordsystem.json files, since they go hand in hand for EEg/iEEG.

Reply:
I think you're using the expression "just kidding" in a very unusual way :) It actually means that you were just joking around, but I believe what you're intending to say it that you wanted to correct yourself?

Anyway, no, this should **not** be changed. According to the specs, there can be separate `electrodes.tsv` files for each run:
https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/03-electroencephalography.html#electrodes-description-_electrodestsv

Reply:
Ah yes I tend to abuse the english language a lot with colloquialisms :p.

Reply:
https://github.com/mne-tools/mne-bids/pull/675#issuecomment-757061083